Dockerfile

```diff
@@ -43,13 +43,10 @@ RUN case ${TARGETPLATFORM} in \
     printf "Building OAuth2 Proxy for arch ${GOARCH}\n" && \
     GOARCH=${GOARCH} VERSION=${VERSION} make build && touch jwt_signing_key.pem
 
-# Copy binary to alpine
+# Copy binary to distroless
 FROM ${RUNTIME_IMAGE}
```

Comment:
`RUNTIME_IMAGE` is currently still overwritten inside the `Makefile`. Needs to be changed:

```
DOCKER_BUILD_RUNTIME_IMAGE    ?= alpine:3.18
DOCKER_BUILDX_ARGS            ?= --build-arg RUNTIME_IMAGE=${DOCKER_BUILD_RUNTIME_IMAGE}
```

Reply:
Fixed